onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -233,6 +233,45 @@ void MultiHeadAttentionTypeAndShapeInference(ONNX_NAMESPACE::InferenceContext& c
   }
 }
 
+void GroupQueryAttentionTypeAndShapeInference(ONNX_NAMESPACE::InferenceContext& ctx) {
```

Comment:
This function is not needed: just propagate both type and shape from input 0 to output 0 is enough.